### ISSUE TITLE
Search from web hoogle

### ISSUE THREAD

**Issue by SeungheonOh** [luc-tielen/telescope_hoogle#7]
Can it be used to search from web hosted hoogle?

**Comment by luc-tielen** [comment]
Yes, that's what it does already: https://github.com/luc-tielen/telescope_hoogle/blob/master/lua/telescope/_extensions/hoogle/live_hoogle_search.lua#L20
Take a look at the URL in the browser gif in the README.

It requires you to have `hoogle` installed, but that searches online, not locally.

**Comment by SeungheonOh** [comment]
Neat. Thank you. Can you set a custom Hoogle source? I have https://hoogle.nix.dance that I want to use.

**Comment by luc-tielen** [comment]
This is all I could find about the `hoogle` cli tool: https://wiki.haskell.org/Hoogle#Command_Line_Search_Flags.

I only make use of that, so the capabilities of the plugin depend on what hoogle offers..

**Comment by amano-kenji** [comment]
hoogle cli tool searches only the local database. I was able to use hoogle CLI tool without network connection. How can I switch between web hoogle and local hoogle?

**Comment by luc-tielen** [comment]
I think it only works locally? https://wiki.haskell.org/Hoogle#Command_Line_Search_Flags doesn't mention anything like that.
